deploy/cluster_role.yaml

```diff
@@ -22,6 +22,8 @@ rules:
       - get
       - delete
       - list
+      - patch
+      - update
```

Comment:
These operations were added to support possible update of oauthclient? In which situation this can happen?

Reply:
I encountered this requirement during some tests I ran, but cannot remember precisely which one. FTR the initial rights Eugene added were `*` ;-)